### PR TITLE
#162045684 Enable travel team members verify requests

### PR DESCRIPTION
**Pull request by Stylll**
#### What does this PR do?
Provides user interface for travel team members to view approved requests and verify them

#### Description of Task to be completed?
- travel team members can view list of verified and approved requests for their centers
- travel team members can see a modal with a button to click to verify a request
- requester gets an in-app notification stating the request has been verified

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- checkout the branch `git checkout ft-verify-request-162045684`
- Start the server with `make start`
- Clone the backend repo `git clone https://github.com/andela/travel_tool_back.git`
- checkout the branch `git checkout ft-verify-request-162045684`
- Start the server with `make start`
- Visit https://travela-local.andela.com
- Login to the application
- Ensure you have a travel team member and manager privilege
- Create a request and approve it.
- Fill the required checklist items.
- Navigate to the My Verifications page under the Request menu on the left side nav.
- You should see a page with a list of requests pending verification.
- Click on a request to review and verify it

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162045684](https://www.pivotaltracker.com/story/show/162045684)

#### Screenshots (if appropriate)
verify-requests

screen shot 2018-11-26 at 1 30 12 pm

### CODE REVIEW

File: src/components/LeftSideBar/Metadata/index.js [hunk 1]
```diff
@@ -40,6 +40,11 @@ const NavItemsMetadata = [
       {
         link_to: '/requests/my-approvals',
         text: 'My Approvals'
+      },
+      {
+        onlyVisibleTo: ['Travel Administrator', 'Super Administrator'],
```
Comment (amarachukwu-agbo) [inline src/components/LeftSideBar/Metadata/index.js]: Is this link not supposed to be visible to travel team members too?

**Comment by amarachukwu-agbo** [comment]
@Stylll, the story says that only requests that are 100% travel ready should be displayed on the `Verifications` page but that is not the case as all approved requests are being displayed
screenshot 2018-11-26 at 4 39 43 pm

**Comment by Stylll** [comment]
> @Stylll, the story says that only requests that are 100% travel ready should be displayed on the `Verifications` page but that is not the case as all approved requests are being displayed
> screenshot 2018-11-26 at 4 39 43 pm

The story's acceptance criteria have been updated. The travel team member can see a request once it is approved by the manager regardless of the travel completion percentage. This is because:
1. It provides the travel team member a list of requests that have been approved so he/she can have a filtered down list of those to reach out to in order to prompt completion where necessary.
2. In a case where a requester has completed his travel checklist and an additional submission is added to the checklist bringing the percentage down to less than 100%, the travel team member can see such requests and notify the requester about the necessary submissions.

File: src/components/LeftSideBar/Metadata/index.js (hunk 1 above)
Comment (Stylll) [reply src/components/LeftSideBar/Metadata/index.js]: @amarachukwu-agbo Thanks for pointing this out. I have updated the array list to include Travel Team Members

**Comment by amarachukwu-agbo** [comment]
Okay @Stylll.

**Pull request merged**